packages/components/organisms/f-checkout/src/tenants/en-GB.js

```diff
@@ -89,13 +89,19 @@ const messages = {
             LOCATION_UNDELIVERABLE: {
                 title: 'The restaurant doesn\'t deliver to this address',
                 message: 'Search for a different restaurant or change to a collection order.',
-                buttonText: 'OK'
+                buttonText: 'Ok'
             },
 
             DEFAULT_CHECKOUT_ISSUE: {
                 title: 'Something went wrong',
                 message: 'Please try again',
                 buttonText: 'Retry'
+            },
+
+            DuplicateOrder: {
+                title: 'This looks like a duplicate order',
+                message: 'So you\'re not charged twice, your order has not been processed',
```

Comment:
You should use a proper apostrophe `’` here – that would save you needing to escape the base `dumb` apostrophe.

Base vertical apostrophes are just known as `dumb apostrophes` ([more info here](https://www.fastcompany.com/3018353/be-smart-dont-use-dumb-quote-marks) if you're a type nerd like me) and it looks nicer to use the proper ones where we remember to.

Reply:
The only problem I have with this is...I need to find out how to type that apostrophe on a Spanish keyboard 🤦

Reply:
I think we should be consistent on apostrophe usages.